plugins/rikki/heroeslounge/models/Playoff.php

```diff
@@ -47,10 +49,8 @@ public function getLongTitleAttribute()
     public function createMatches()
     {
         $playoff = $this;
-        $timezone = 'Europe/Berlin';
+        $timezone = TimezoneHelper::defaultTimezone();
         if ($this->type == 'playoffv1') {
-            //TODO configure dates and timezone from backend
```

Comment:
should line 52 be `getTimezone()` now?

Reply:
I believe this requires a lot more work than just updating the timezone variable, as the automatic match creation is currently hard-coded, and manually changed by Zalz for every playoff creation

Reply:
You can leave this function as is for now, its a different (large) issue. Last time I ran it, the $timezone parameter was ignored anyway.

Reply:
Should I leave the TODO?  Should I not change the timezone?

Reply:
Indeed, its part of #35